### ISSUE TITLE
An Error Occurred after Saving first Content Type

### ISSUE THREAD

**Issue by mrtunes** [strapi/strapi#7508]
**Describe the bug**
A small red error prompt after hitting save on a new content type. An Error Occurred

**Steps to reproduce the behavior**

When I run Yarn Develop I can make a content type (music), and then I add a field for Media and call it Mp3. When I hit save on this it says An Error Occurred (small popup). 

**Expected behavior**
The content type would be created after hitting save

**Screenshots**
![image](https://user-images.githubusercontent.com/1292976/90504420-086fc580-e11f-11ea-9670-948925f417ae.png)

**Code snippets**
If applicable, add code samples to help explain your problem.

**System**

- Node.js version: 12.18.3
- NPM version: 6.14.6
- Strapi version: 3.1.4
- Database:
- Operating system:

**Additional context**
The log shows: 

```
[2020-08-18T10:41:20.268Z] debug GET /admin/3.c30723bd.chunk.js (2 ms) 200
[2020-08-18T10:41:20.287Z] debug GET /admin/bd03a2cc277bbbc338d464e679fe9942.woff2 (9 ms) 200
[2020-08-18T10:41:20.288Z] debug GET /admin/8b4f872c5de19974857328d06d3fe48f.woff2 (8 ms) 200
[2020-08-18T10:41:20.318Z] debug GET /admin/cccb897485813c7c256901dbca54ecf2.woff2 (8 ms) 200
[2020-08-18T10:41:20.320Z] debug GET /admin/33d5f0d956f3fc30bc51f81047a2c47d.woff2 (9 ms) 200
[2020-08-18T10:41:20.512Z] debug GET /admin/c500da19d776384ba69573ae6fe274e7.woff2 (2 ms) 200
[2020-08-18T10:44:06.859Z] debug POST /content-type-builder/content-types (40 ms) 400

```

**Comment by derrickmehaffy** [comment]
> This is a templated message

Hello, please follow the issue template.

A proper issue submission let's us better understand the origin of your bug and therefore help you.

I will reopen your issue when we receive the issue following the template guidelines and properly fill out the template. You can see the template guidelines for bug reports [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

Please update the issue.

Thank you.

> A copy of the bug report template can be found below:

```
<!--
Hello 👋 Thank you for submitting an issue.

Before you start, please make sure your issue is understandable and reproducible.
To make your issue readable make sure you use valid Markdown syntax.

https://guides.github.com/features/mastering-markdown/

Please ensure you have also read and understand the contributing guide.

https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue
-->

**Describe the bug**
A clear and concise description of what the bug is.

**Steps to reproduce the behavior**

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Code snippets**
If applicable, add code samples to help explain your problem.

**System**

- Node.js version: <!-- Please ensure you are using the Node LTS version (v12) -->
- NPM version:
- Strapi version: <!-- Beta and Alpha versions are no longer supported -->
- Database:
- Operating system:

**Additional context**
Add any other context about the problem here.
```

**Comment by derrickmehaffy** [comment]
Can you please add any of your browser dev tools logs (F12 menu in chrome) so we can see the content of the 400 error message.

**Comment by mrtunes** [comment]
![image](https://user-images.githubusercontent.com/1292976/90561870-56141e80-e16f-11ea-9164-46d8b84c3097.png)

**Comment by derrickmehaffy** [comment]
Can you provide a bit more detail from those logs, with the current information I can't reproduce your issue

**Comment by mrtunes** [comment]
> Can you provide a bit more detail from those logs, with the current information I can't reproduce your issue

yep no problem - i should be expanding the logs in the developer tools screenshot above, right (hit more triangles)?

**Comment by derrickmehaffy** [comment]
In the network tab when you make the request you should be able to see the full error message instead of just the console.

**Comment by mrtunes** [comment]
ok this line turned red - 
{error: {contentType.name: [,…]}}
error: {contentType.name: [,…]}
contentType.name: [,…]

This field was media and it was called mp3, so I tried also adding a field as simple text and calling it title, and i get the same error. I tried it one more time and this error appeared as well

![image](https://user-images.githubusercontent.com/1292976/90657968-01c17b00-e211-11ea-8fb9-28aa9b8e5e4b.png)

**Comment by derrickmehaffy** [comment]
I haven't been able to reproduce this issue :confused: @lauriejim thoughts?

**Comment by lauriejim** [comment]
Hello! Can you please provide the 2 information missing in the issue template:
- Database:
- Operating system:
Question, did you create your application or it's a migration.
And can you provide clear steps to reproduce this issue?
Thank you.

**Comment by mrtunes** [comment]
I'll fill those in - it's SQLite because I ran a quickstart, and i'm on Mac 10.15.5

**Comment by chrisk8er** [comment]
i have same issue...

**Comment by derrickmehaffy** [comment]
I've tested a few times now in various ways (Linux, Windows, Mac and different node versions/databases) and still cannot reproduce this.

We need some additional steps from either of you that can try to help us track down the cause because right now, we can't reproduce it and thus can't locate where the problem might be.

**Comment by derrickmehaffy** [comment]
Closing as no new information provided

**Comment by FlacorLopes** [comment]
I'm getting the same problem with the version 3.4.5.
When I create a content-type named Music the content type builder does not let me save it.

I must rename its display name for something else and save it.

It happens when creating the first content-type also, but haven't tried it differently.

![image](https://user-images.githubusercontent.com/1523957/106217798-cdd18500-61b4-11eb-9b50-ccb47e9ef818.png)

**Comment by derrickmehaffy** [comment]
@FlacorLopes can you please open a new issue?

**Comment by TacticCoder** [comment]
Same for me

**Comment by derrickmehaffy** [comment]
See new issue: https://github.com/strapi/strapi/issues/9266
